package.json

```diff
@@ -12,8 +12,8 @@
     "doc": "doc",
     "test": "tests"
   },
-  "main": "addon/@ember/test-helpers/index.ts",
-  "types": "addon/@ember/test-helpers/index.d.ts",
```

Comment:
Noticed this bug, which may have contributed to users not being able to correctly reference types in TypeScript.